<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ATMECSINLT-744" failures="0" tests="4" name="com.atmecs.api.get.testscripts.TC_01_Get_No_Auth" time="40.444" errors="0" timestamp="28 May 2020 15:05:57 GMT" skipped="0">
  <testcase classname="com.atmecs.api.get.testscripts.TC_01_Get_No_Auth" name="getMaxResults" time="27.916"/>
  <testcase classname="com.atmecs.api.get.testscripts.TC_01_Get_No_Auth" name="getMissingQueryError" time="2.679"/>
  <testcase classname="com.atmecs.api.get.testscripts.TC_01_Get_No_Auth" name="GetInvalidRangeError" time="2.879"/>
  <testcase classname="com.atmecs.api.get.testscripts.TC_01_Get_No_Auth" name="getBookCount" time="6.970"/>
</testsuite> <!-- com.atmecs.api.get.testscripts.TC_01_Get_No_Auth -->
